README.md

```diff
@@ -5,3 +5,19 @@ Repository for running MLIR compiled stuff on SNAX
 ## Requirements
 
 [stardew](https://github.com/Groverkss/stardew)
```

Comment:
Just to clarify, when you meant this is the requirement, you meant we also have to follow the instructions from stardew right?

Reply:
This is from a previous PR, but yes @jorendumoulin is going to add some more instructions on that later